lib/brainstem/cli/generate_api_docs_command.rb

```diff
@@ -164,10 +141,57 @@ def option_parser
             options[:builder][:args_for_atlas][:controller_matches].push(matcher)
           end
 
-
           opts.on('--markdown', 'use markdown format') do |o|
             options[:sink][:options][:format] = :markdown
           end
+
+          opts.on('-m', '--multifile-presenters-and-controllers',
+            'dumps presenters and controllers to separate files (default)') do |o|
+            if options[:sink][:options][:format] == :oas_v2
+              raise NotImplementedError.new("Multi File support for Open Api Specification is not supported yet")
+            else
+              options[:sink][:method] = Brainstem::ApiDocs::Sinks::ControllerPresenterMultifileSink.method(:new)
+            end
+          end
+
+          opts.on('--output-extension=EXTENSION', 'extension that should be used for output files') do |extension|
+            options[:sink][:options][:output_extension] = extension
+          end
+
+          #########################################################
+          #                                                       #
+          # Open Api Specification generation specific commands:  #
+          #                                                       #
+          #########################################################
+
+          # Future proofing for different Open Api Specification versions.
+          opts.on('--open-api-specification=VERSION',
+                  'dumps an Open Api Specification for presenters and controllers in a single file') do |oas_version|
+            case oas_version.to_i
+              when 2
+                options[:sink][:options][:format] = :oas_v2
+              else
+                raise NotImplementedError.new(
+                  "Please specify the version of Open Api Specification to be generated e.g. --open-api-specification=2"
+                )
+            end
+            options[:sink][:method] = Brainstem::ApiDocs::Sinks::OpenApiSpecificationSink.method(:new)
+          end
+
+          opts.on('--api-version=API_VERSION',
+                  'sets the version of the generated documentation') do |api_version|
+            options[:sink][:options][:api_version] = api_version
+          end
+
+          opts.on('--ignore-tagging',
+                  'does not add the tag definitions in the Open Api Specification') do |api_version|
```

Comment:
Looks like we're inconsistent with 

> Open Api Specification

versus

> Open Api specification

I guess the former is preferred given the OAS acronym?